scan.sh

```diff
@@ -74,8 +80,14 @@ main() {
         opts=$(parse_args "$@" || exit 1)
 
         python3 cs_scanimage.py $opts
+        EXIT_CODE=$?
+        echo "::set-output name=exit-code::$EXIT_CODE"
+        exit $EXIT_CODE
     else
         python3 cs_scanimage.py
+        EXIT_CODE=$?
+        echo "::set-output name=exit-code::$EXIT_CODE"
+        exit $EXIT_CODE
```

Comment:
Totally forgot, but why did we add this again?

Also, I think we can condense the exit code handling to after the if statement so that we aren't duplicating code.

Reply:
This allows the action to provide the exit code back to the pipeline so that it could potentially be used for subsequent steps. Without this the pipeline is blind to the failure. If we are manipulating exit codes for particular "findings" I think it's best that we provide those back to the pipeline. Also I think it's a good practice to provide output for the action if you can. I fooled with the exit code handling for quite some time when I worked on this IIRC and this was the "best" working option I could come up with here.